api/task.go

```diff
@@ -68,9 +68,19 @@ const (
 type TaskDatabasePITRRestorePayload struct {
 	// The project owning the database.
 	ProjectID int `json:"projectId,omitempty"`
+
+	// TargetInstance must be within the same environment as the instance of the original database.
```

Comment:
Should it be within the same project?

Reply:
The new database should be under the same project as the original database. Instance and project have no strong relationship.